src/OpenSpaceToolkit/Astrodynamics/Trajectory/Sequence.cpp

```diff
@@ -233,6 +233,11 @@ Sequence::Solution Sequence::solve(const State& aState) const
     {
         for (const Segment& segment : segments_)
         {
+            if (segment.accessEventCondition()->targetIsRelative())
+            {
+                segment.accessEventCondition()->updateTarget(initialState);
```

Comment:
```suggestion
                segment.accessEventCondition()->setTargetState(initialState);
```

nit: Seeing it in context, maybe this naming? It seems like this is something that each segment will have to set once at the beginning, so this naming feels a little cleaner to me.